The orca startup shell script seems to contain some *isms.
Remove solaris specific PATH components, use set -e, and other cleanups.

--- orca-2.22.1/src/orca/orca.in.orig	2008-03-08 13:27:37.000000000 +0100
+++ orca-2.22.1/src/orca/orca.in	2008-04-08 20:33:58.000000000 +0200
@@ -30,7 +30,7 @@
 
 # Set the user's $PATH for this script.
 #
-export PATH="${PATH}:/usr/ccs/bin:/usr/bin:/usr/sbin:/bin:/usr/sfw/bin:/usr/openwin/bin:/usr/X11R6/bin"
+export PATH="${PATH}:/usr/bin:/usr/sbin:/bin:/usr/X11R6/bin"
 
 # If you set RUNONCE to "true", then this will just run Orca once and quit.
 #
@@ -55,16 +55,8 @@
 # do not want a background process that periodically pings Orca to see
 # if it is responding.
 #
-IFS=:
-WGETCMD=
-WATCHDOG=0
-for dir in $PATH:/usr/sfw/bin:/usr/local/bin; do
-    test -x "$dir/wget" && {
-        WGETCMD="$dir/wget"
-        WATCHDOG=1
-        break
-    }
-done
+WATCHDOG=1
+WGETCMD=`which wget || WATCHDOG=0`
 
 # Cleans up any orca-related processes that might be running,
 # restricting it to those processes owned by the user. These include
@@ -258,7 +250,7 @@
 then
     main
 else
-    if [ `grep -c "\-q" <<< $ARGS` -gt 0 ]
+    if echo "$ARGS" | grep -- "-q" > /dev/null
     then
         cleanup
     else
@@ -270,7 +262,7 @@
         # erroneously types an illegal command line argument, the
         # help text is emitted and the other orca is not killed.
         #
-        if [ `egrep -c "\-s|\-g|\-t|\-n|\-u|\-e|\-d" <<< $ARGS` -eq 0 ]
+        if ! echo "$ARGS" | egrep -- "\-s|\-g|\-t|\-n|\-u|\-e|\-d" > /dev/null
         then
             runOrca "NO_CLEANUP"
         else
